lua/Network.lua

```diff
@@ -672,18 +673,27 @@ metaTableNetwork_ = {
 -- maximum error in the distance up to the its value.
 -- Therefore, the default value for this argument is zero.
 -- @arg data.lines A base::CellularSpace with lines to create network. It can be for example a set of roads.
--- @arg data.outside User-defined function that computes the distance based on an
--- Euclidean distance to enter and to leave the Network.
+-- @arg data.outside User-defined function that converts the distance based on an
+-- Euclidean distance to a distance in the geographical space. This function is
+-- applied to enter and to leave the network, as well as to try to see whether
+-- the distance without using the network is shorter than using the network.
 -- If not set a function, will return the distance itself.
+-- This function gets one argument with the distance in Eucldean space
+-- and must return the distance in the geographical space.
 -- @arg data.progress Optional boolean value indicating whether Network will print messages
 -- while processing values. The default value is true.
--- @arg data.strategy Strategy to be used in the network (optional).
 -- @arg data.target A base::CellularSpace with the destinations of the network.
--- @arg data.weight User defined function to change the network distance.
+-- @arg data.inside User defined function that converts the distance based on
+-- an Euclidean distance to a distance in the geographical space. This function
+-- is applied to every path within the network.
 -- If not set a function, will return the distance itself. Note that,
```

Comment:
Much better changed to `inside`.
Currently, `inside` and `outside` are mandatory, but, I agree that they could be optional.